packages/itwinui-react/src/core/LabeledInput/LabeledInput.tsx

```diff
@@ -110,8 +109,8 @@ export const LabeledInput = React.forwardRef((props, ref) => {
       )}
 
       <Box
-        className={cx('iui-input-with-icon', inputWrapperProps?.className)}
         {...inputWrapperProps}
+        className={cx('iui-input-with-icon', inputWrapperProps?.className)}
```

Comment:
nit: This should already be handled in #1777, so maybe revert it to prevent conflicts (plus it's unrelated to this PR).